nav-tracking-mitigations/stateful-client-bounce.sub.https.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE HTML>
+<meta charset=utf-8>
+<title>Bounce Tracking Mitigations: Stateful Client Bounce</title>
+<body>body for test_driver.bless</body>
+<script src=/resources/testharness.js></script>
+<script src=/resources/testharnessreport.js></script>
+<script src=/resources/testdriver.js></script>
+<script src=/resources/testdriver-vendor.js></script>
+<script>
+  promise_setup(async () => {
+    try {
+      await test_driver.set_storage_access('*', '*', 'blocked');
+    } catch (e) {
+      // Ignore, can be unimplemented if the platform blocks cross-site
+      // cookies by default. If this failed without default blocking the test
+      // will fail.
+    }
+  });
+
+  promise_test(async t => {
+    const {promise: getLinkPromise, resolve: getLinkResolver} =
+      Promise.withResolvers();
```

Comment:
This seems overly complicated, can't you just await the load / link getter and then continue? Is there a specific reason why you need to resolve a promise externally?